React/tic-tac/src/localStorage.js

```diff
@@ -0,0 +1,21 @@
+export const loadState = () => {
```

Comment:
I'd recommend moving this file to the `services` folder since it will handle localStorage actions. You can name it something like `localStorageService` so it will be easier to find and reference whenever you need it.